spectacles/cli.py

```diff
@@ -157,10 +158,11 @@ def wrapper(*args, **kwargs):
 def set_file_handler(directory: str) -> None:
 
     global LOG_FILEPATH
+    global LOG_DIRECTORY
 
-    log_directory = Path(directory)
-    LOG_FILEPATH = Path(log_directory / LOG_FILENAME)
-    log_directory.mkdir(exist_ok=True)
+    LOG_DIRECTORY = Path(directory)
+    LOG_FILEPATH = Path(LOG_DIRECTORY / LOG_FILENAME)
```

Comment:
Don't need to wrap this in Path anymore, can just do `LOG_DIRECTORY / LOG_FILENAME` since the directory is already a Path object.